doc/whats_new/v0.21.rst

```diff
@@ -71,6 +71,11 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.cluster`
 ......................
 
+- |API| Deprecate ``n_components_`` attribute of :class:`cluster.AgglomerativeClustering`
```

Comment:
We usually put API at the end of the list

Reply:
Maybe just say "the n_components_ attribute in ... has been renamed to n_connected_components_"

Reply:
Understood, addressed this in commit [#e2a7e4a](https://github.com/scikit-learn/scikit-learn/pull/13427/commits/e2a7e4a9487b365fb789a577f6a99f94bcdb92eb)